rpm/tayga.spec

```diff
@@ -5,8 +5,18 @@ Summary:	Simple, no-fuss NAT64
 License:	GPLv2+
 URL:		http://www.litech.org/%{name}/
 Source0:	http://www.litech.org/%{name}/%{name}-%{version}.tar.bz2
-Patch0:		tayga-0.9.2_cflags_override.patch
-Patch1:		tayga-c99.patch
+Patch1:		0001-use-var-spool.patch
+Patch2:		0002-manpage.patch
+Patch3:		0003-cflags-override.patch
+Patch4:		0004-quote-make-var.patch
+Patch5:		0005-guard-chdir.patch
+Patch6:		0006-guard-write.patch
+Patch7:		0007-static-EAM.patch
+Patch8:		0008-manpage-RFC.patch
+Patch9:		0009-systemd-service-file-for-tayga.patch
```

Comment:
We should ignore this patch as we are not running tayga as a service. ConnMan runs the binary when needed and systemd use would just confuse (at least the current approach).